react/Popper/testing.js

```diff
@@ -0,0 +1,23 @@
+// Fixes the error "TypeError: document.createRange is not a function" in jest,
+// see https://github.com/mui-org/material-ui/issues/15726#issuecomment-493124813
+const fixPopperTesting = () => {
+  let createRangeBackup
+
+  beforeAll(() => {
+    createRangeBackup = global.document.createRange
+    global.document.createRange = jest.fn(() => ({
+      setStart: () => {},
+      setEnd: () => {},
+      commonAncestorContainer: {
+        nodeName: 'BODY',
+        ownerDocument: document
+      }
+    }))
+  })
+
+  afterAll(() => {
+    global.document.createRange = createRangeBackup
+  })
+}
+
+export default fixPopperTesting
```

Comment:
Nit: since this file could have other functions (helping to interact with a Popper from the test), I think it would be better to have a normal export instead of a default one.

As a user, I would try `import { fixPopperTesting } from './testing'` since the file is named `testing` and not `fixPopperTesting.js`